management-core/src/main/java/com/ns/task/service/ProductCoreServiceImpl.java

```diff
@@ -38,15 +47,23 @@ public ProductCoreServiceImpl(ModelMapper mapper, ProductRepository repository)
                 .collect(Collectors.toList());
     }
 
+    @RabbitListener(queues = "q.management.insert")
+    public ProductDTO receiverRPC(ProductDTO product) {
+        logger.info("Received message from RabbitMQ: {}", product.toString());
+        return insertProduct(product);
+    }
+
     @Override
     public ProductDTO insertProduct(ProductDTO product) {
         ProductEntity productEntity = convertToEntity(product);
         try {
             logger.debug("Sending data to DB {}", productEntity);
             productEntity = repository.saveProduct(productEntity);
+            product.setMessage("Name and model of the service exists already");
```

Comment:
Why setting this message here?

Reply:
You are right, my mistake sorry I'll be more careful